dashboard/cron.py

```diff
@@ -417,6 +417,15 @@ def update_resource_access(self):
             logger.debug(f'resource_access_df:\n'
                          f'{resource_access_df}\n'
                          f'{resource_access_df.dtypes}')
+            
+            # only keep access events generated by students
+            student_enrollment_df = pd.read_sql(
+                'select user_id, course_id '
+                f'from user where enrollment_type=\'StudentEnrollment\'',
+                engine)
+            resource_access_df = pd.merge(
+                resource_access_df, student_enrollment_df,
+                on=['user_id', 'course_id'], how='inner')
 
```

Comment:
This change seems to work well with new courses added to MyLA DB. But for the old courses, it doesn't remove the TA or TeacherEnrollment info. I think that is by design we adopted long back not deleting all the live events data for saying cost. 

To me this approach work as long we still have the filter in the view.py only sending the student access in the response.

Reply:
We can do `update course set data_last_updated=null` to force full refresh of resource access events. It would lead to extra BigQuery cost, but the amount should be reasonable.

Reply:
I agree with this approach if it doesn't cost much with the 8T/byte quota limit we have. This kind of saves @jennlove-um since she often tests from old test courses. 

or we could a manual delete of all the resources access info with Teacher and TA.

Reply:
This seems like it would work. The `inner` approach is a bit less readable than say a `pandas` filter, but it's less code. A comment might be appropriate.

Reply:
added comment for inner merge

Reply:
Yeah, I agree with @pushyamig that clearing out the resource access events would make sense, even if it adds cost. That may need to be in the release notes too.